coralproject/talk
# Moderator Community

## What does this PR do?

Allows moderators to access the community pages.

## How do I test this PR?

1. Login as a moderator user. (Note, you can easily add the moderator role to an existing user via the cli: `./bin/cli -c .env users addrole <userID> <role>` where the user id is found from `./bin/cli users list`
2. View the community page.
3. See it work!
